### PR TITLE
add global phase rerank count

### PR DESCRIPTION
**Pull request by arnej27959**
this follow the pattern from MatchPhase properties.
@bratseth please review
@havardpe FYI

### CODE REVIEW

**Review by geirst — COMMENTED** [review]
Consider adding a test in container-search/src/test/java/com/yahoo/search/test/QueryTestCase.java, similar to how "trace.profiling..." is tested.

File: container-search/src/main/java/com/yahoo/search/ranking/GlobalPhaseRanker.java [hunk 1]
```diff
@@ -101,4 +106,12 @@ private Optional<GlobalPhaseSetup> globalPhaseSetupFor(Query query, String schem
                 .flatMap(evaluator -> evaluator.getGlobalPhaseSetup(query.getRanking().getProfile()));
     }
 
+    private int resolveRerankCount(GlobalPhaseSetup setup, Query query) {
+        if (setup == null) return 0;
```
Comment (bratseth) [inline container-search/src/main/java/com/yahoo/search/ranking/GlobalPhaseRanker.java:110]: Should this be after the next line?

**Review by bratseth — COMMENTED** [review]
LGTM, and yes I think we should have a test.

File: container-search/src/main/java/com/yahoo/search/ranking/GlobalPhaseRanker.java (hunk 1 above)
Comment (arnej27959) [reply container-search/src/main/java/com/yahoo/search/ranking/GlobalPhaseRanker.java]: no (added comment to clarify)

**Review by bratseth — APPROVED** [review]

**Pull request merged**